### PR TITLE
Port to Python 2 & 3, Django 1.8+ and pytest

### PR DESCRIPTION
**Pull request by EnTeQuAk**
Part of https://github.com/mozilla/addons-server/issues/4997

### CODE REVIEW

File: aesfield/field.py [hunk 1]
```diff
@@ -23,50 +22,46 @@ def __init__(self, *args, **kwargs):
         self.aes_key = kwargs.pop('aes_key', '')
         super(AESField, self).__init__(*args, **kwargs)
 
+    def deconstruct(self):
+        name, path, args, kwargs = super(HandField, self).deconstruct()
+        # TODO: Should `aes_method` be present in the serialized form too?
```
Comment (eviljeff) [inline aesfield/field.py]: Any chance of getting an answer to this before merging the pull?  I'm loathed to add more TODOs any any code.

Comment (EnTeQuAk) [inline aesfield/field.py]: heh good point. I think I simply forgot to look that up, will do that before merging.

Comment (EnTeQuAk) [inline aesfield/field.py]: Fixed in  	63ab373

**Comment by EnTeQuAk** [comment]
@andymckay mind giving me access to pypi so that I can release a new version?

File: aesfield/field.py (hunk 1 above)
Comment (eviljeff) [inline aesfield/field.py]: Umm, I don't see you actually removing the `TODO...` line

Comment (EnTeQuAk) [inline aesfield/field.py]: doh, super fail… I just fixed it, that's embarrassing

**Pull request merged**